### PR TITLE
Peter users

### PR DESCRIPTION
**Pull request by r05al**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<h4><%= @user.username %></h4>
+
+<div>
+  <img src="<%= @user.avatar %>">
+</div>
+<div data-role="follow">
+  <button data-role="new-follow" value="<%= @user %>">follow</button>
```
Comment (jkrth617) [inline app/views/users/show.html.erb:7]: I am wondering why you have a data-role on both the container and the button itself. Do you have a click button on the button and then you want to hid the div after a click? You might not need a container for this? I duno :confounded:

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+</div>
+<div data-role="unfollow" class="hidden">
+  <button data-role="unfollow">unfollow</button>
```
Comment (jkrth617) [inline app/views/users/show.html.erb:10]: sam as above :arrow_double_up:

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+<div class="followers">
+  <% @user.followers.each do |follower| %>
+    <%= render partial: 'user', locals: { user: follower } %>
+  <% end %>
+</div>
+
+<h5><%= @user.followees.count %> following</h5>
+<div class="followees">
+  <% @user.followees.each do |followee| %>
+    <%= render partial: 'user', locals: { user: followee } %>
```
Comment (jkrth617) [inline app/views/users/show.html.erb:22]: would this mean that if I go and click on a user to view their profile I would be able to see a list of everyone they follow? :cookie:

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  <% end %>
+</div>
+
+<h5><%= pluralize(@user.drops.count, 'drops') %></h5>
+<div class="drops">
+  <% @user.drops.each do |drop| %>
+    <%= render partial: 'drop', locals: { drop: drop } %>
```
Comment (jkrth617) [inline app/views/users/show.html.erb:29]: talked about this. you know wassup, only the author of the posts should see this not people viewing their profile :floppy_disk:

File: app/views/users/show.html.erb (hunk 1 above)
Comment (davidgtu) [inline app/views/users/show.html.erb:7]: jake is a meanie. you do you peter.

**Pull request closed without merging**